zoho-objects/zoho-results-small.ts

```diff
@@ -0,0 +1,1654 @@
+export const results: any = [{
```

Comment:
Is this file needed?

Reply:
It was just for testing. I can get rid of it.